### ISSUE TITLE
Chat view: show accessibility help does not work when focus is on request item

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#27988]
Refs: https://github-com/microsoft/vscode/-/186619

1. Open chat view. Ask some things
2. Move keyboard focus on the request item (what you typed)
3. Press `Alt + F2` -> nothing happens 🐛 

This is inconsistent with the behaviour when the focus is on the response.

Megan informed me that this behaviour is by design, but I do not see understand what is the benefit of this behaviour. What are we getting with this inconsistency?
